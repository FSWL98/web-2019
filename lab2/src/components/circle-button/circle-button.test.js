import React from "react";
import CircleButton from "./circle-button";
import renderer from 'react-test-renderer';

it('CircleButton disable', () => {
    const component = renderer.create(<CircleButton isDisabled={true} />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});

it('CircleButton default', () => {
    const component = renderer.create(<CircleButton />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
