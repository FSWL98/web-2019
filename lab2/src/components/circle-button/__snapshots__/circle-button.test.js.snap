// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CircleButton default 1`] = `
<div
  className=" d-flex justify-content-end p-0 m-0"
>
  <button
    className="circle-button"
    disabled={false}
    type="button"
  >
    X
  </button>
</div>
`;

exports[`CircleButton disable 1`] = `
<div
  className=" d-flex justify-content-end p-0 m-0"
>
  <button
    className="circle-button"
    disabled={true}
    type="button"
  >
    X
  </button>
</div>
`;
